scripts/ci_integration_tests.sh

```diff
@@ -4,10 +4,11 @@ set -xeuo pipefail
 script_dir=$(dirname ${BASH_SOURCE[0]})
 source ${script_dir}/ci_build_common.sh
 
-enclaveMode=$1
-runtimeType=$2
+# Set parameters to lowercase
+enclaveMode=${1,,}
+runtimeType=${2,,}
```

Comment:
To the best way to make sure this works is check the values are valid.

Reply:
Do we want to check the values are valid in the script? It's only TC and us using it, so I don't expect having completely wrong input. But don't mind adding the check if required.

Reply:
Well, @filipesoliveira had to make some TC changes in the last week or so, so I think it's worth it if it's straightforward to do.

Reply:
Done, PTAL.